plugin/main.py

```diff
@@ -4,13 +4,12 @@
 from .steam import Steam, SteamLibraryNotFound, SteamExecutableNotFound
 
 from flox import Flox, ICON_SETTINGS
-from flox.string_matcher import string_matcher
+from flox.string_matcher import string_matcher, DEFAULT_QUERY_SEARCH_PRECISION, QUERY_SEARCH_PRECISION
 
 
 class SteamSearch(Flox):
-
+    
     def query(self, query):
-        self.logger
```

Comment:
Hahaha, I was wondering why this was here.  I was really hoping there weren't side effects of the property access somehow.

Reply:
Probably got distracted and forgot! >_<